### ISSUE TITLE
will not open in mac Monterey

### ISSUE THREAD

**Issue by austenlennon** [OpenShot/openshot-qt#4437]
**Describe the bug**
A clear and concise description of the issue

**Steps to reproduce the behavior:**

**Expected behavior**

**System Details**
- OpenShot Version: 2.6.1-dev
- libopenshot Version:  0.2.7-dev
- Platform: Darwin-21.1.0-x86_64-i386-64bit
- Processor: i386
- Machine: x86_64
- Python version: 3.7.11
- Qt5 version: 5.15.0
- PyQt5 version: 5.15.1.dev2007111314
- Qt Detected Languages: 
- LANG Environment Variable: 
- LOCALE Environment Variable: 
- Daily Build: Verified issue still exists in daily build: http://github.com/OpenShot/openshot-qt/releases/download/OpenShot-v2.6.1-dev-daily-8292-21597c8a-73c519f0-x86_64.dmg

**Log Files**
- [openshot-qt.log](https://s3.amazonaws.com/logs.openshot.org/mra0eq0uuk5r6ysxl2oaik2o4geyqgta/openshot-qt.zip) (2 KB)
- [libopenshot.log](https://s3.amazonaws.com/logs.openshot.org/mra0eq0uuk5r6ysxl2oaik2o4geyqgta/libopenshot.zip) (2 KB)

**Exception / Stacktrace**
```No stacktrace found in log files```

**Screenshots** (Optional)
If applicable, add screenshots to help explain your problem. You can include screenshots by
copy/pasting them on GitHub or dragging-and-dropping into the GitHub page. All images are public,
so please don't post screenshots containing personal information.

**Comment by dvago** [comment]
Same problem here, seems like the latest daily build as well as previous versions aren't compatible with the latest Mac os Monterey.

**Comment by didiercapozzi** [comment]
Same problem here. Impossible to open it

**Comment by shopshire** [comment]
The latest version is still 2.6.1 and does not launch in Monterey.

**Comment by chrisvire** [comment]
I tried installing the latest release version and it doesn't launch. I get these errors in console.app:

```
error	09:19:27.029236-0400	kernel	Sandbox: logd_helper(159) deny(1) file-read-data /Applications/OpenShot Video Editor.app/Contents/MacOS/lib/PyQt5/QtCore.so
error	09:19:27.182643-0400	kernel	Sandbox: logd_helper(159) deny(1) file-read-data /Applications/OpenShot Video Editor.app/Contents/MacOS/lib/PyQt5/QtWidgets.so
error	09:19:27.182871-0400	kernel	Sandbox: logd_helper(159) deny(1) file-read-data /Applications/OpenShot Video Editor.app/Contents/MacOS/lib/PyQt5/QtGui.so
error	09:19:27.183037-0400	kernel	Sandbox: logd_helper(159) deny(1) file-read-data /Applications/OpenShot Video Editor.app/Contents/MacOS/lib/PyQt5/QtWebChannel.so
error	09:19:27.183218-0400	kernel	Sandbox: logd_helper(159) deny(1) file-read-data /Applications/OpenShot Video Editor.app/Contents/MacOS/lib/PyQt5/QtNetwork.so
error	09:19:27.183367-0400	kernel	Sandbox: logd_helper(159) deny(1) file-read-data /Applications/OpenShot Video Editor.app/Contents/MacOS/lib/PyQt5/QtPrintSupport.so

```

I tried the latest daily build (June 22, 2022) and it was able to launch.

**Comment by randomhammer** [comment]
The latest build will not work for me. I am unable to launch the application.  I get the following entries in my log

error	21:56:24.592813-0500	kernel	Sandbox: logd_helper(589) deny(1) file-read-data /Applications/OpenShot Video Editor.app/Contents/MacOS/lib/PyQt5/QtWidgets.so
error	21:56:24.592940-0500	kernel	Sandbox: logd_helper(589) deny(1) file-read-data /Applications/OpenShot Video Editor.app/Contents/MacOS/lib/PyQt5/QtGui.so
error	21:56:24.593034-0500	kernel	Sandbox: logd_helper(589) deny(1) file-read-data /Applications/OpenShot Video Editor.app/Contents/MacOS/lib/PyQt5/QtWebChannel.so
error	21:56:24.593133-0500	kernel	Sandbox: logd_helper(589) deny(1) file-read-data /Applications/OpenShot Video Editor.app/Contents/MacOS/lib/PyQt5/QtNetwork.so
error	21:56:24.593225-0500	kernel	Sandbox: logd_helper(589) deny(1) file-read-data /Applications/OpenShot Video Editor.app/Contents/MacOS/lib/PyQt5/QtPrintSupport.so

There appear to be permissions errors in even the latest build, at least the one I keep getting when I hit download.

**Comment by jhays** [comment]
I still get this issue, macOS 12.5.1, app v2.6.1.
App finishes verifying, attempts to launch (icon bounces in mac's dock), but never actually opens any windows. Immediately closes and disappears from the dock.

**Comment by Colorjet3** [comment]
Startup issues have been resolved with the latest DEV Daily Build v3.0.0.
